Beyond09-SOW.md

```diff
@@ -0,0 +1,508 @@
+
+* Team 1: Yftach Herzog, Yuval Vinokur, Nitzan Mordechai, Sebastian Scheinkman
+* Team 2: Kobi Hakimi, Moshe Sheena, Or Mergi, Ygal Blum
+* Team 3: Yariv Rachmani, Benny Rochwerger, Amos Mastbaum, Daniel Keler
+* Team 4: Nati Friedman, Omer Amsalem, Noam Finkelstein, Alicia Petrunin, Galit Rosenthal
+
+## Project ideas
+
+You can consider below ideas or think up your own.
+
+* Dating website
+* Drop shipping web store - selling other people's products on your website
+* Personal password maintenance application
+* Course certificate generation and distribution platform
+* An online riddle or math game
+
+Please avoid the following ideas as they have already been implemented too many
+time in the past by Beyond students:
+
+* Platform for finding part-time jobs for students.
+* Platform for academic courses and grading.
+* Platform for finding/recruiting jobs/employess.
+* Platform for for finding roomates or appartments.
+* Platform for freelancers to offer their services.
+* P2P Lending service
+
+__Note:__ The project used as a learning platform, it should not be
+ unique or amazing, it could defeat learning process and goals.
+
+## Syllabus (list of sessions)
+
+| #    | Date                | Topic(s)                           | Lecturer        | Location        |
+|------|---------------------|------------------------------------|-----------------|-----------------|
+|-     |Wed, Feb 19, 2023    |First challenge hand-off and student selection (30m pre-course session)|Barak Korren|Online/Remote|
+|**-** |**Fri, Mar 03, 2023**|**First challenge submission date** |**_(Self work)_**|                 |
+|1     |Sun, Mar 05, 2023    |General introduction                |Barak Korren     |MTA College      |
+|      |                     |Project Architecture overview       |Barak Korren     |MTA College      |
+|      |                     |Second challenge hand-off           |Barak Korren     |MTA College      |
+|**-** |**Fri, Mar 10, 2023**|**Second challenge submission date**|**_(Self work)_**|                 |
+|2     |Sun, Mar 12, 2023    |Introduction to Python              |Barak Korren     |Online/Remote    |
+|3     |Sun, Mar 19, 2023    |Databases, data model and the ORM   |Omer Amsalem     |Online/Remote    |
+|      |                     |Project teams announcement          |                 |                 |
+|**4** |**Sun, Mar 26, 2023**|**Demo 1: Development environment** |Barak            |Online/Remote    |
+|-     |Sun, Apr 02, 2023    |_(Passover - College is closed)_    |-                |                 |
+|-     |Sun, Apr 09, 2023    |_(Passover - College is closed)_    |-                |                 |
+|5     |Sun, Apr 16, 2023    |Business logic and Testing          |Eddy Haas        |Online/Remote    |
+|**6** |**Sun, Apr 23, 2023**|**Demo 2: Data Model and business logic**|**TBD**     |**Online/Remote**|
+|-     |**Apr 24 - Apr 30**  |**Performance review week**         |**-**            |**-**            |
```

Comment:
Just take into account that we have:
25.4 - Yom HaZikaron
26.4 - Yom HaAtzmaut